app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -24,8 +24,10 @@ def days_remaining
       end
 
       def formatted_due_date
-        enrollment.due_date.in_time_zone(USPS_SERVER_TIMEZONE).
-          strftime(I18n.t('time.formats.event_date'))
+        I18n.l(
```

Comment:
Sorta curious why this had to change from `strftime` to `I18n.l`, or was that more of a readability improvement / consistent change?

Reply:
That previous version wasn't displaying the months with the correct translated month name. The `I18n.l` or localize method will correctly translate the month names when formatting the date.